data/post-marcxml-curl.md

```diff
@@ -0,0 +1,10 @@
+# How to post marcxml record using curl via access token
+
+You can post marcxml record via an access token using the api documents as follows:
+assumening that your token is: my_token and the xml record is saved in document.xml
```

Comment:
Maybe add a link, or even just a mention, to the part where you've documented how to generate the token.